File: Python/pe001.py
#Author - Jamie Woods
#https://projecteuler.net/problem=1


import math

def main():
    n = 1000
    sum = 0
    
    for x in range(1, n):
        if((x % 3) == 0 or (x % 5) == 0):
            sum += x
        
    print sum
    
if __name__ == "__main__":
    main()

File: C#/CSharpSolutions/Solutions/pe001.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CSharpSolutions.Solutions
{
    //https://projecteuler.net/problem=1

    public class pe001 : ISolution
    {
        const int LIMIT = 1000;

        public void Run()
        {
            var sum = 0;

            for (int count1 = 0; count1 < LIMIT; count1++)
            {
                if (count1 % 3 == 0 || count1 % 5 == 0)
                {
                    sum += count1;
                }
            }

            Console.WriteLine($"The solution to problem 001 is {sum}");
        }

    }
}
